calour/dsfdr.py

```diff
@@ -229,79 +247,82 @@ def dsfdr(data, labels, method='meandiff', transform='rankdata', alpha=0.1, nump
             sample_nonzero = sample_nonzero - np.mean(sample_nonzero)
             label_nonzero = label_nonzero - np.mean(label_nonzero)
             tstat[i] = np.dot(sample_nonzero, label_nonzero)
-            t[i] = np.abs(tstat[i])
+            t[i]=np.abs(tstat[i])
 
             permlabels = np.zeros([len(label_nonzero), numperm])
             for cperm in range(numperm):
-                rlabels = np.random.permutation(label_nonzero)
-                permlabels[:, cperm] = rlabels
+                rlabels=np.random.permutation(label_nonzero)
+                permlabels[:,cperm] = rlabels
```

Comment:
missing space after ',' (again annoying pep8 needs it otherwise it complains...)